ocfweb/account/templates/account/vhost/index.html

```diff
@@ -4,33 +4,42 @@
 {% block content %}
     <div class="ocf-content-block">
         <p>
-            One of the benefits of hosting your student group's site with the OCF is that you have the option of using an address like "mygroup.berkeley.edu" to access your site.
-        However, in order to obtain a berkeley.edu name for your site, you must meet both university policies regarding acceptable use of berkeley.edu names, as well as OCF policies for virtual hosting.
+            One of the benefits of hosting your website with the OCF is that department and student groups have the option of using an address like "mygroup.berkeley.edu" to access their site.
+            However, in order to use a custom domain name for your site, you must meet both university policies regarding acceptable use of berkeley.edu names, as well as OCF policies for virtual hosting.
         </p>
         <p>
             Please start by reading the
             <a href="{% url 'doc' 'services/vhost' %}">OCF Virtual Hosting documentation</a>.
-            If you believe your group satisfies all requirements listed, you
-            may complete the form below to request for us to set up a virtual
-            host for your account.
+            If you believe you have satisfied all requirements listed, you may
+            complete the form below to request for us to set up a virtual host
+            for your account.
         </p>
 
         <p>
             You are currently logged in using the OCF account
-            <strong>{{user}}</strong>. If this is not the group you wish to
+            <strong>{{user}}</strong>. If this is not the account you wish to
             request a virtual hosting for, please <a href="{% url 'logout' %}">
             log out</a> and log back in.
         </p>
 
         <div class="well">
             <h3 style="margin-top: 0;">Virtual Hosting Request Form</h3>
             <ul>
-                <li><strong>Group Name:</strong> {{attrs.cn.0}}</li>
-                <li><strong>Group OCF Account:</strong> {{user}}</li>
-                <li>
-                    <strong>Current OCF-Hosted Group Website:</strong>
-                    <a href="{{group_url}}">{{group_url}}</a>
-                </li>
+                {% if is_group %}
+                    <li><strong>Group Name:</strong> {{attrs.cn.0}}</li>
+                    <li><strong>Group OCF Account:</strong> {{user}}</li>
+                    <li>
+                        <strong>Current OCF-Hosted Group Website:</strong>
+                        <a href="{{group_url}}">{{group_url}}</a>
+                    </li>
+                {% else %}
+                    <li><strong>Name:</strong> {{attrs.cn.0}}</li>
+                    <li><strong>OCF Account:</strong> {{user}}</li>
+                    <li>
+                        <strong>Current OCF-Hosted Website:</strong>
+                        <a href="{{group_url}}">{{group_url}}</a>
+                    </li>
+                {% endif %}
```

Comment:
I think we do so people requesting a vhost for their group don't put _their_ name at the top, scroll down, then realize it was supposed to be the _group_ name at the top.

Speaking of, I just realized the form now asks faculty for their name twice (oops). Thanks for inadvertently calling my attention to it.